packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -145,10 +145,23 @@ function ApplicationDetails() {
     setIsReadmeOpen(false);
   };
 
+  if (isGuideView) {
+    return (
+      <GuideModal
+        guideId={guideId}
+        app={data}
+        onClose={(id) => {
+          navigate(`/applications/${id}`);
+        }}
+      />
+    );
+  }
+
   return (
     <DetailsPage
       backLink="/applications"
       backLinkTitle="application_details.back_to_applications"
+      // Hide app details loading skeleton if it's loading guide
```

Comment:
this comment stands out but seems not matching the code below

Reply:
I remember I removed it, don't know why it came back... Will remove again